### ISSUE TITLE
Availability in google hosted libraries

### ISSUE THREAD

**Issue by akkumar** [angular/material#1659]
The latest version seems to be 0.8.1  . 

Can this be made available at google hosted libraries herewith: https://developers.google.com/speed/libraries/devguide#angular_material  ? The latest version in that cdn happens to be 0.7.1 though . 

Feel free to let me know if this involves any process / patch as such - in case this needs help. Thanks !

**Comment by thorsenty** [comment]
Also eager to start using 0.8.x in my app.

**Comment by davidshen84** [comment]
**[0.8.2](https://ajax.googleapis.com/ajax/libs/angular_material/0.8.2/angular-material.min.js)** is available at google cdn, but the page is not updated to include this version.

**Comment by ScottRFrost** [comment]
Google hosted libraries is now 4 versions behind (0.9.0 vs 0.9.4)

**Comment by jaGarcia** [comment]
Hosted libraries is at 1.0.0 with release at 1.0.3.  Perhaps this bug can be about automating the push of new versions to the hosted libraries cdn.

tl;dr +1

**Comment by ScottRFrost** [comment]
https://gitcdn.xyz/ is down hard (DNS won't even resolve) but http://www.jsdelivr.com/ is up to date at 1.0.3

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.
